### PR TITLE
UI-4471 upgrade the ancillary packages

### PR DESCRIPTION
**Pull request by sheela-svmx**
Updated the ancillary packages in CRA

### CODE REVIEW

File: packages/react-scripts/template/.template.dependencies.json [hunk 1]
```diff
@@ -5,10 +5,10 @@
     "express": "^4.16.2",
     "fetch-mock": "^6.5.2",
     "lightning-container": "^1.0.0",
-    "lodash": "4.17.11",
-    "morgan": "^1.9.1",
+    "lodash": "4.17.14",
```
Comment (joshsylvester) [inline packages/react-scripts/template/.template.dependencies.json:8]: `lodash` should be updated to `4.17.15` - this version causes a npm audit error to show, and running `npm audit fix` updates to `4.17.15`

Comment (sheela-svmx) [reply packages/react-scripts/template/.template.dependencies.json:8]: OOh is it? let me fix that

**Comment by sheela-svmx** [comment]
@joshsylvester updated

**Comment by svmx-pratima** [comment]
2 approvals-merging

**Pull request merged**